src/entries/RollForVolume/RollForVolume.js

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useRef, useState } from "react"
+import { ClipLoader } from "react-spinners"
+import styled, { keyframes } from "styled-components"
+
+import BaseEntry from "../../components/BaseEntry/BaseEntry"
+
+const diceFaces = [
+  "https://upload.wikimedia.org/wikipedia/commons/c/c5/Dice-0.png",
+  "https://upload.wikimedia.org/wikipedia/commons/2/2c/Alea_1.png",
+  "https://upload.wikimedia.org/wikipedia/commons/b/b8/Alea_2.png",
+  "https://upload.wikimedia.org/wikipedia/commons/2/2f/Alea_3.png",
+  "https://upload.wikimedia.org/wikipedia/commons/8/8d/Alea_4.png",
+  "https://upload.wikimedia.org/wikipedia/commons/5/55/Alea_5.png",
+  "https://upload.wikimedia.org/wikipedia/commons/f/f4/Alea_6.png",
+]
+
+const appearAnimation = keyframes`
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+`
+
+const Content = styled.div`
+  display: flex;
+  flex-basis: 100%;
+  min-height: 100%;
+  background: #f5f5f5
+    url("https://images.unsplash.com/photo-1519744346361-7a029b427a59")
+    no-repeat center;
+  background-size: cover;
+  align-items: center;
+  justify-content: center;
+  animation: ${appearAnimation} 0.5s ease;
+`
+
+const Placeholder = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+`
+
+const DiceContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const DiceImage = styled.img`
+  width: 50px;
+  height: 50px;
+`
+const Row = styled.div`
+  display: flex;
+  width: 60%;
+  height: 80%;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+`
+
+const TotalVolume = styled.h2`
+  color: white;
+  font-family: "Merriweather Sans";
+  text-align: center;
+  font-size: 2rem;
+`
+
+const PressContainer = styled.div`
+  font-family: "Merriweather Sans";
+  font-size: 3rem;
+  font-weight: 400;
+  margin: 0 auto;
+  color: white;
+  text-shadow: 2px 5px 5px #222222;
+  animation: ${appearAnimation} 0.5s ease;
+
+  @media (min-width: 50rem) {
+    font-size: 5rem;
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+
+const RollForVolume = () => {
+  const [assetsLoaded, setAssetsLoaded] = useState(0)
+  const [assetsError, setAssetsError] = useState(false)
+  const [chillTime, setChillTime] = useState(false)
+  const [diceValues, setDiceValues] = useState(Array(16).fill(6))
+  const audioGainRef = useRef()
+  const audioContextRef = useRef()
+  const completeAssets = assetsLoaded > 1
+
+  const playSound = async () => {
+    if (!audioContextRef.current) {
+      const audioContext = new AudioContext()
```

Comment:
I've updated the sample to include a cleanup function. you might want to do that as well since navigating away or if the component unmounts the audio is still working.

Reply:
Done! I also set the `audioSource` to loop the music until the user navigates to another entry component.